hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/schemas/V0490TokenSchema.java

```diff
@@ -49,7 +49,7 @@ public class V0490TokenSchema extends Schema {
     private static final long MAX_STAKING_INFOS = 1_000_000L;
     private static final long MAX_TOKENS = 1_000_000_000L;
     private static final long MAX_ACCOUNTS = 1_000_000_000L;
-    private static final long MAX_TOKEN_RELS = 1_000_000_000L;
+    private static final long MAX_TOKEN_RELS = 4_000_000_000L;
```

Comment:
This change will not affect existing states, since the number of HalfDiskHashMap buckets is stored in MerkleDb metadata, so the new size hint will be ignored. Path to hash and path to KV indices will have 4B capacity, so we will be able to store up to 4B token rels, but real memory consumption will be much less till there are really 4B entities